styles.css

```diff
@@ -0,0 +1,120 @@
+.roboto-regular {
+    font-family: "Roboto", sans-serif;
+    font-weight: 400;
+    font-style: normal;
+}
+
+.ticket-container {
+    margin: 50px 0 0 50px;
+    width: 782px;
+    height: 273px;
+    background: #EFC684;
+    border-radius: 10px;
+    display: flex;
+}
+
+.main-section {
+    width: 289px;
+}
+
+.ticket-text {
+    text-transform: uppercase;
+    font-size: 55px;
+    width: 206px;
+    height: 128px;
+    margin: 26px 0 0 20px;
+}
+
+.ticket-details {
+    display: flex;
+    justify-content: flex-start;
+}
+
+.ticket-details div {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.ticket-category {
+    width: 82px;
+    height: 44px;
+    margin: 36px 30px 39px 27px;
+    border: solid 1px black;
+    font-size: 20px;
+    justify-content: center;
+    text-transform: uppercase;
+}
+
+.ticket-price {
+    width: 82px;
+    height: 44px;
+    margin-top: 36px;
+    margin-left: 10px;
+    margin-bottom: 39px;
+    border: solid 1px black;
+    font-size: 20px;
+    justify-content: center;
+}
+
+.movie-details {
+    display: flex;
+    flex-direction: column;
+    margin-right: 52px;
+    justify-content: space-around;
+}
+
+.movie-hour {
+    margin-top: 26px;
+    width: 85px;
+    height: 23px;
+}
+
+.movie-localisation {
+    display: flex;
+    justify-content: space-between;
+}
+
+.gate {
+    width: 82px;
+    height: 23px;
+}
+
+.seat {
+    width: 92px;
+    height: 23px;
```

Comment:
Rely on the content to expand its parent instead of providing the height and width explicitly if you can

Reply:
done